### PR TITLE
HV-1086 - Update dependencies

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/browse/HV-1086

### CODE REVIEW

File: engine/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/JavaFXClassLoadingTest.java [hunk 1]
```diff
@@ -14,16 +14,15 @@
 import org.fest.assertions.Assertions;
 import org.hibernate.validator.internal.util.privilegedactions.LoadClass;
 import org.hibernate.validator.testutil.TestForIssue;
-
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.classloader.ShrinkWrapClassLoader;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
-import org.junit.Test;
+import org.testng.annotations.Test;
```
Comment (gunnarmorling) [inline engine/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/JavaFXClassLoadingTest.java]: Good catch!

File: README.md [hunk 2]
```diff
@@ -42,14 +42,9 @@ Logging will delegate any log requests to that provider.
       In an EE environment they are often already provided.
 
         <dependency>
-           <groupId>javax.el</groupId>
-           <artifactId>javax.el-api</artifactId>
-           <version>2.2.4</version>
-        </dependency>
-        <dependency>
```
Comment (gunnarmorling) [inline README.md]: So they don't have two distinct artifacts for API and implementation any more?

File: pom.xml [hunk 3]
```diff
@@ -81,24 +81,49 @@
...
-        <!-- Used to add further arguments to the arg line for specific profiles -->
-        <surefire.argLine.extension></surefire.argLine.extension>
+        <!-- Don't upgrade chronos to 1.1.0 yet. Dependencies get copied to wrong location -->
+        <org.codehaus.mojo.chronos.version>1.0-beta-3</org.codehaus.mojo.chronos.version>
+
+        <!--
+            These dependencies should be aligned with the ones from the Wildfly version we support
+            See http://search.maven.org/#search|gav|1|g%3A"org.wildfly"%20AND%20a%3A"wildfly-parent"
+        -->
+        <classmate.version>1.3.1</classmate.version>
```
Comment (gunnarmorling) [inline pom.xml]: Is this the one used in WF (10.1)? Same question for the other deps just in case.

File: pom.xml [hunk 4]
```diff
@@ -81,24 +81,49 @@
...
+        <jsoup.version>1.8.3</jsoup.version>
+        <joda-time.version>2.7</joda-time.version>
+        <slf4j.version>1.7.7</slf4j.version> <!-- Note: Wildfly 10.1.0.CR1 uses a custom jboss version -->
```
Comment (gunnarmorling) [inline pom.xml]: You are mentioning CR1 here, but further below it's using the Final?

**Comment by gunnarmorling** [comment]
Nice clean-up; added some minor questions inline.

File: README.md (hunk 2 above)
Comment (gsmet) [inline README.md]: Nope. JBoss separated them in WildFly. They published the extracted API on Central but not the extracted impl. So I chose to use the official Glassfish jar which seems to be the norm out there.

File: pom.xml (hunk 3 above)
Comment (gsmet) [inline pom.xml]: Did you take a look at the comment just above :). Would have been perverse to add the comment and not align them :).

File: pom.xml (hunk 4 above)
Comment (gsmet) [inline pom.xml]: Mmh, copy/pasto from my work on OGM, will fix.

**Comment by gsmet** [comment]
Rebased, fixed the comment about slf4j and added a commit to update the developers part of the parent pom.

File: pom.xml (hunk 3 above)
Comment (gunnarmorling) [inline pom.xml]: Ok :)

**Pull request merged**